docs/account/sso-login/okta-saml.md

```diff
@@ -70,6 +76,42 @@ Pleas be aware that, enabling SSO for **APPCIRCLE LOGIN** doesn't enable SSO for
 
 <Screenshot url='https://cdn.appcircle.io/docs/assets/oktasamlsettings3-new.png' />
 
+## Okta Managing User Groups
+
```

Comment:
Overview is missing. Why should users want to manager user groups from okta?
We can refer to the store behaviour (beta access none by default, live access all by default).